tests/scripts/custodian/Dockerfile

```diff
@@ -5,39 +5,47 @@ ARG AWS_ACCESS_KEY_ID
 ENV AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
 ARG AWS_SECRET_ACCESS_KEY
 ENV AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
-ARG AZURE_SUBSCRIPTION_ID
-ENV AZURE_SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID
+ARG AZURE_AKS_SUBSCRIPTION_ID
+ENV AZURE_SUBSCRIPTION_ID=$AZURE_AKS_SUBSCRIPTION_ID
 ARG AZURE_TENANT_ID
 ENV AZURE_TENANT_ID=$AZURE_TENANT_ID
 ARG AZURE_CLIENT_ID
 ENV AZURE_CLIENT_ID=$AZURE_CLIENT_ID
 ARG AZURE_CLIENT_SECRET
 ENV AZURE_CLIENT_SECRET=$AZURE_CLIENT_SECRET
-ARG RANCHER_LINODE_CREDENTIAL
-ENV RANCHER_LINODE_CREDENTIAL=$RANCHER_LINODE_CREDENTIAL
+ARG RANCHER_LINODE_ACCESSKEY
+ENV RANCHER_LINODE_CREDENTIAL=$RANCHER_LINODE_ACCESSKEY
+
 ARG OVERRIDE_REGION
 ENV OVERRIDE_REGION=$OVERRIDE_REGION
+
 ARG USER_KEYS
 ENV USER_KEYS=$USER_KEYS
 ARG DONOTDELETE_KEYS
 ENV DONOTDELETE_KEYS=$DONOTDELETE_KEYS
+
 ENV CUSTODIAN_YAML=aws.yaml
 
+ENV GOOGLE_APPLICATION_CREDENTIALS=google_credentials.json
+ENV GOOGLE_CLOUD_PROJECT=ei-container-platform-qa
+
 ADD ./* ./
 SHELL ["/bin/bash", "-c"] 
 
 RUN apt-get update && apt-get install -y python3 python3-pip python3-venv apt-transport-https ca-certificates gnupg curl
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-cli -y
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
-RUN python3.11 -m venv custodian-venv
+
+RUN python3 -m venv custodian-venv
```

Comment:
I doubt it, but leaving comment to be sure. We don't think there is going to be an issue with a minor version of `python3` that conflicts, correct?

Reply:
correct. In fact, adding the minor version was causing an issue 😓